datascroller/scroller.py

```diff
@@ -499,7 +499,7 @@ def key_press_and_print_df(stdscr, df):
             if len(query_string) > 0:
                 try:
                     df_window = df_window.filter(query_string, viewing_area)
-                except pandasql.sqldf.PandaSQLException:  # TODO better exception handling
+                except sqldf.PandaSQLException:  # TODO better exception handling
                     pass
                     # TODO(johncmerfeld): Reprimand the user?
```

Comment:
maybe just printing that there was an empty query entered?

Reply:
That might be better handling. My modification is just to remove the reference to the pandasql module. The import statement is: from pandasql import sqldf, so if you try to reference pandasql, you get a ModuleNotFound error.

Reply:
Ah okay, I'm trying out `scroll_demo` now and I see that an empty query is not a problem. If I make a mistake in the query, on the other hand, I'm kicked out of the scroller. Not a big deal but it might be a nicer user experience to write the error message for a few seconds and then have it disappear. This could be slated for a later feature as well.

Message I get if I type "/asdf":
```
pandasql.sqldf.PandaSQLException: (sqlite3.OperationalError) near "asdf": syntax error
[SQL: asdf]
(Background on this error at: http://sqlalche.me/e/e3q8)
```

Reply:
Good catch. I think for this whole thing to be really primetime ready, it shouldn't crash out no matter what the user does, and this could be another patched hole. I like the idea of just printing the error. We could also do a little parsing (of the error, not the query) and print a slightly more helpful message than the one they give.